Source/Reporters/FileAppendingReporter.jsl

```diff
@@ -15,8 +15,9 @@ Define Class(
 	
 	filepath = "";
 	
-	_init_ = Method({filepath},
+	_init_ = Method({filepath, write successes=0},
```

Comment:
Remove `=0` default argument. The default is already in the factory function below, which should always be used to create this reporter.